source/process/support.md

```diff
@@ -61,4 +61,31 @@ Typically found in `/etc/nginx/nginx.conf` and `/etc/nginx/sites-available/matte
 #### Can you send us a snippet of your Nginx error logs around the time of the incident?
 Typically found in `/var/log/nginx/error.log`
 
+## LDAP Issues
 
+#### Attach your LDAP settings from config.json
+The username/password should be removed.
+
+#### What AD/LDAP server and version are you using?
+For example Active Directory on Windows Server 2016
+
+#### Are there any LDAP errors in the logs?
+
+#### Is there a limit on the number of users the LDAP server can return in a single query? If so, have you set the Maximum Page Size in Mattermost?
+Many LDAP servers have an upper limit on the number of users returned. So they might be hitting that limit. An error will appear in the logs usually informing of this case, but it's good to try anyway. 
+
+#### Can you send an example user from you system? ldapsearch command format is preferred.
+
```

Comment:
### Please provide the full User Filter if you are having LDAP Synchronisation issues.
   
+ The user filter should be : (&(objectCategory=Person)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))

Reply:
Based on conversation with Eval User https://pre-release.mattermost.com/core/pl/o4ibswjbctbcmfttdbroxx4z6y.